### ISSUE TITLE
Support for 0.7.0?

### ISSUE THREAD

**Issue by zetas** [julianlam/nodebb-plugin-support-forum#3]
I'm not sure if this is an issue because I'm on 0.7 but any user can view the support forum and see a list of topics. If they try to click any topics but their own they'll get a "bad route" error which I image is correct but they shouldn't be able to see the topic list or the most recent reply (which is displayed by default in the persona theme and can't be disabled via the ACP afaik). 

In the meantime I'll have to disable this plugin as It doesn't seem to be doing what it's supposed to.

**Comment by julianlam** [comment]
HI David, 

You're correct in that you're not supposed to be able to see the recent replies in the category view. Sounds like a bug.

**Comment by julianlam** [comment]
However, being able to see the topic list was by design, so users wouldn't come to an empty category even if it is active.

**Comment by zetas** [comment]
Hmm, the users can give away private details in the topic header if they think it can't be seen by everyone. Wouldn't it make sense to only display their own topics and empty otherwise? It seems like a weird usecase to have a list of topics you can't browse and when you try to you get a route error instead of an error that explains that you don't have access because it's a special type of category.

Here's a screenshot of the list:
![screenshot 2015-05-10 12 15 02](https://cloud.githubusercontent.com/assets/1036810/7555072/35971316-f70e-11e4-9194-81f12bc17cd5.png)

And when I try to view the topic:
![screenshot 2015-05-10 12 15 40](https://cloud.githubusercontent.com/assets/1036810/7555075/504d5440-f70e-11e4-89e3-3bb7ec0a1d19.png)

**Comment by julianlam** [comment]
Right. the Access Denied error is correct, but the recent reply being shown
is not.

**Comment by jasonphillips** [comment]
For the issue of users seeing posts by others on the category page, I've submitted a pull request that makes this an optional behavior (something I needed as well).

https://github.com/julianlam/nodebb-plugin-support-forum/pull/4
